reference/CloudPay-API-Specification.yaml

```diff
@@ -2299,6 +2318,12 @@ paths:
           name: apisessionid
           required: false
           description: When SSO is not active the purchase URL is pointed at the RegisterCreateAccount webpage
+          - schema:
+            type: string
+          in: query
+          name: entitlement
+          required: false
+          description: Used to return a list of packages that match a single or multiple entitlements.
```

Comment:
seems to be breaking the doco because of incorrect indentation

<img width="297" alt="Screenshot 2024-03-21 at 11 43 13" src="https://github.com/StreamAMG/cloudpay-documentation-portal/assets/149777948/06b586c1-f92a-4a32-ba48-cb255aadae32">

Reply:
resolved